### DISCUSSION TITLE
Delay when using getUserTimeline(...)

### DISCUSSION THREAD

**Discussion by Productscore-Apps** [redouane59/twittered#176] [Q&A] [answered]
Hi

I noticed that the method TwitterClient::getUserTimeline(...) has a delay of roughly 8-9 seconds in comparison to a JavaScript Twitter library I have been using (this library uses the old v1 endpoints though).

Are you aware of this? 
If so, do you think it is a problem of your library or is there maybe a twitter related performance issue with the v2 endpoints?

I am using twittered version 1.21 at the moment.

Thanks and Regards

Raphael

**Comment by redouane59** [comment, marked as answer]
Hi @Productscore-Apps ,
Thanks for the feedback, I wasn't aware about it. Can be interesting to ask the question to someone form twitter because on my side I cannot explain how it could be different.

I'm going to ping @suhemparack now

**Reply by redouane59** [reply]
This is maybe the solution : 
![image](https://user-images.githubusercontent.com/4500524/116596369-2c291380-a91c-11eb-91d2-a3456293a0f4.png)

See : https://twittercommunity.com/t/get-user-timeline-performances-v1-vs-v2/153484

**Reply by Productscore-Apps** [reply]
Hi @redouane59 
Interesting chart. That's probably it. Thank you.
